layouts/index.html

```diff
@@ -30,14 +30,15 @@ <h1 class="bold-title {{ if .Site.Params.fadeIn | default true }}fade-in one{{ e
                     </div>
                 </div>
                 <!-- Hero body title end -->
+                {{ end }}
+                <div class="{{ if or (.Site.Params.fadeIn | default true) .Site.Params.fadeInIndex }}fade-in three{{ end }}">
+                    {{ partial "nav.html" . }}
+                </div>
             </div>
```

Comment:
Why are this changes needed?

Reply:
I'm not sure I get the question, it's motivated by having the nav bar part of the top section so it gets moved in the corresponding div.